cmd/go-swagger/restapi/operations/path_traversal/path_traversasl_front.go

```diff
@@ -0,0 +1,56 @@
+// Code generated by go-swagger; DO NOT EDIT.
```

Comment:
`traversal` is misspelled in the file name. Likewise for 3 other files.

Reply:
corrected